features/provide_vat_receipt.feature

```diff
@@ -0,0 +1,33 @@
+@US259
+Feature: Add VAT receipt at the bottom of the printed summary page
+As a business customer
+I want to retain a VAT receipt for all of my purchases
+So that I can claim back my VAT
+
+Acceptance criteria:
+
+1) Must contain:
+    - Unique invoice number that follows on from the last invoice
+    - LR business name and address
+    - LR VAT number
+    - The tax point (or ‘time of supply’) if this is different from the invoice date [see activity point below for more info on this point]
+    - Description of the goods or services
+    - Rate of VAT charged per item - if an item is exempt or zero-rated make clear no VAT on these items
```

Comment:
Strictly speaking, VAT **is** charged in the "zero-rated" case (I believe) but is zero!